config/zones/MN.yaml

```diff
@@ -46,6 +49,9 @@ emissionFactors:
         datetime: '2020-01-01'
         source: IEA 2020; assumes 92.85% coal, 5.78%% oil, 1.37% hydro in unknown
         value: 799.2
+      - datetime: '2024-11-25'
+        source: IEA 2020; assumes 94.14% coal, 5.86% oil in unknown
```

Comment:
This `source` also says `IEA 2020` but has different values from two lines up, is that correct?

Reply:
Yes, basically I used the same source but simply removed hydro since that will now be reported on it's own.